README.md

```diff
@@ -12,85 +12,101 @@
 - [Sass vs SCSS](#sass-vs-scss)
 - [Frameworks](#frameworks)
 - [Libraries and Mixins](#libraries-and-mixins)
+  - [Animation](#animation)
+  - [Color](#color)
   - [Grid](#grid)
+  - [Media Queries](#mediaqueries)
+  - [Misc](#Misc)
```

Comment:
Can fix this line to be `[Misc](#misc)` instead? 

It's supposed to be small `m` after `#`, or else the anchor link won't work 😢